py-polars/Cargo.toml

```
@@ -19,11 +19,11 @@ bincode = "1.3"
 # todo: unfix when compilation problem is solved
 libc = "0.2"
 ndarray = "0.15"
-numpy = "0.16"
+numpy = "0.18"
```

Similarly: must this include the bug-fix number? I.e. must it be:
```suggestion
numpy = "0.18.0"
```